Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
-3: The Electors shall meet in their respective States, and vote by Ballot for two Persons, of whom one at least shall not be an Inhabitant of the same State with themselves. And they shall make a List of all the Persons voted for, and of the Number of Votes for each; which List they shall sign and certify, and transmit sealed to the Seat of the Government of the United States, directed to the President of the Senate. The President of the Senate shall, in the Presence of the Senate and House of Representatives, open all the Certificates, and the Votes shall then be counted. The Person having the greatest Number of Votes shall be the President, if such Number be a Majority of the whole Number of Electors appointed; and if there be more than one who have such Majority, and have an equal Number of Votes, then the House of Representatives shall immediately chuse by Ballot one of them for President; and if no Person have a Majority, then from the five highest on the List the said House shall in like Manner chuse the President. But in chusing the President, the Votes shall be taken by States, the Representation from each State having one Vote; A quorum for this Purpose shall consist of a Member or Members from two thirds of the States, and a Majority of all the States shall be necessary to a Choice. In every Case, after the Choice of the President, the Person having the greatest Number of Votes of the Electors shall be the Vice President. But if there should remain two or more who have equal Votes, the Senate shall chuse from them by Ballot the Vice President.8
+5: The Congress may determine, by majority vote in both the House and the Senate, the Time of choosing the Electors, and the Day on which they shall give their Votes; which Day shall be the same throughout these United States. Once the Time and Day have first been selected, they shall be fixed until the election is completed.
 
-4: The Congress may determine the Time of chusing the Electors, and the Day on which they shall give their Votes; which Day shall be the same throughout the United States.
+6: No Person except a natural born Citizen shall be eligible to the Office of President; neither shall any Person be eligible to that Office who shall not have attained to the Age of forty Years, and been twenty Years a Resident within these United States, and who shall not have honorably served a term of no less than four years in the Militia of a State, or in the Military of these United States before the year two thousand and twenty, or in the Navy.
```

Comment:
Here, I not only increase the time-based requirements upon a Presidential candidate, I also add the requirement that he be a military veteran of honorable service. The President's most important role is as the chief officer in international relations, both peaceful and otherwise. He should therefore be someone who was willing to put himself forward in his youth to go into danger himself, and who respects the meaning of that choice, and understands the structure of the military.